opencog/openpsi/utilities.scm

```diff
@@ -326,13 +326,13 @@
 		result
 		; else check if entity is an evaluation or predicate or schema
 		(let ((type (cog-type entity)))
-			(if (or (equal? type 'PredicateNode)
-				   (equal? type 'GroundedPredicateNode)
+			(if (or (equal? type 'GroundedPredicateNode)
 				   (equal? type 'DefinedPredicateNode))
 				(set! result (cog-evaluate! (Evaluation entity (List)))))
-			(if (or (equal? type 'SchemaNode)
-            		(equal? type 'GroundedSchemaNode)
-            	    (equal? type 'DefinedSchemaeNode))
+			(if (equal? type 'PredicateNode)
+				(set! result (cog-tv entity)))
```

Comment:
you need to broaden your definition of "User Error" !!!

On Mon, Aug 1, 2016 at 11:40 AM, Eddie Monroe <notifications@github.com>
wrote:

> In opencog/openpsi/utilities.scm
> <https://github.com/opencog/opencog/pull/2334#discussion_r73010496>:
>
> >  				   (equal? type 'DefinedPredicateNode))
> >  				(set! result (cog-evaluate! (Evaluation entity (List)))))
> > -			(if (or (equal? type 'SchemaNode)
> > -            		(equal? type 'GroundedSchemaNode)
> > -            	    (equal? type 'DefinedSchemaeNode))
> > +			(if (equal? type 'PredicateNode)
> > +				(set! result (cog-tv entity)))
>
> No user error on the part of either programmer I think.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencog/opencog/pull/2334/files/255ac40a46ca72d1fe5b4f88e3959a99a2c6658d#r73010496>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFolXPA9AzfX5p_tGzrisX9SG1KWFy_Dks5qbiGOgaJpZM4JYxpE>
> .
>



-- 
Ben Goertzel, PhD
http://goertzel.org

Super-benevolent super-intelligence is the thought the Global Brain is
currently struggling to form...